insights/client/__init__.py

```diff
@@ -547,6 +547,24 @@ def show_results(self):
             else:
                 raise e
 
+    def show_inventory_deep_link(self):
+        """
+        Show a deep link to this host inventory record
+        """
+        system = self.connection._fetch_system_by_machine_id()
+        if system is not None and system:
```

Comment:
Could you just do `if system:` here since None and False are both falsey values?

Reply:
Ah, yea probably. This is a side-effect of my "writing Python like Go" behavior. I have a habit of first checking for NULL, and then checking for value.